Develop

ChrisIdema opened the pull request:
removed magic numbers and changed initial positions of snake and food to allow smaller widths and heights

ChrisIdema committed:
-removed some magic numbers

Commit:
-magic numbers now properly removed
-fixed bug of swapped x and y

Commit:
-magic numbers now really properly removed
-fixed another bug of swapped x and y

Commit:
-made initial positions of snake and food smaller values of x and y so it fits in a smaller window (8*8)

patrickloeber commented on snake-game/snake.py:
```diff
@@ -1,18 +1,23 @@
+#constants
+
+width = 58
+height = 18 
```
Use 60 and 20 here and adapt the other formulas accordingly. Also for constants use UPPERCASE

patrickloeber reviewed (changes requested):
Thanks for the improvements! I have some minor comments

Commit:
-changed constants from inner window size to outer window size

ChrisIdema commented:
Perhaps you might be interested in my new branch https://github.com/ChrisIdema/python-fun/tree/wide_mode . I've added some interesting features:
- added wide mode option to correct for the fact that the font's height is often almost double its width by adding spaces between "blocks"
- replaced top and bottom window borders with underscore and overline so their margins match side margins more closely. Especially the bottom margin was really annoying as it looked like the snake could drop one more line. I used an extended ascii character for the overline character, so that might not be supported by all terminals.

Might be too much bloat for such a simple project so I didn't create a pull request, but for me it is definitely more playable this way. By replacing the curses functions I was able to get it to run on micropython with a led matrix display too.
